rdmaxcel-sys-hip/README.md

```diff
@@ -1,3 +1,34 @@
+# rdmaxcel-sys-hip
+The folder should mirror the CUDA rdmaxcel-sys implementation and provide AMD/RoCM support.
+
+## Hipify
+The AMD/RoCM implementation include files(e.g. .hip files) which are generated via hipifying original CUDA files(e.g. .cu) files:
```

Comment:
In the final version we should consider hipifying the CUDA implementation on the fly to keep implementations in sync. We can integrate this in the build.rs similar to how build_amd.py works under pytorch.